<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPattern</class>
 <widget class="QDialog" name="SelectPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Pattern</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QTextBrowser" name="path_rute">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="path_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 7;
border : 1px solid black;
background-color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>Change...</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Name">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>NAME</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
